index.html

```diff
@@ -0,0 +1,206 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
```

Comment:
You are missing `<meta http-equiv="x-ua-compatible" content="ie=edge">`, this is still needed until legacy IE's are off the map.